OpenOversight/app/main/forms.py

```diff
@@ -100,3 +100,15 @@ class AssignmentForm(Form):
     unit = QuerySelectField('unit', validators=[Optional()],
                             query_factory=unit_choices)
     star_date = DateField('star_date', validators=[Optional()])
+
+
+class DepartmentForm(Form):
+    name = StringField(
+        'Full name of police department, e.g. Chicago Police Department',
+        default='', validators=[Regexp('\w*'), Length(max=500), DataRequired()]
```

Comment:
Is there a reason `Length` on these don't match the columns? Entries of length 255<x<500 for `name` will throw an exception in OpenOversight/app/main/views.py:add_department()